build.gradle

```diff
@@ -41,6 +41,7 @@ buildscript {
         classpath 'com.google.firebase:firebase-appdistribution-gradle:5.0.0'
         classpath 'com.google.firebase:firebase-crashlytics-gradle:2.9.5'
         classpath "com.diffplug.spotless:spotless-plugin-gradle:7.0.0.BETA1"
+        classpath 'io.javalin:javalin:6.1.4'
```

Comment:
I'm marking this as resolved since this change was reverted per our [other discussion](https://github.com/firebase/firebase-android-sdk/pull/6290#discussion_r1771710582)